### DISCUSSION TITLE
Hardware capabilities

### DISCUSSION THREAD

**Discussion by codepilot** [Xilinx/fpga24_routing_contest#15] [Q&A] [answered]
Will the hardware have avx-512?

Will opencl be exposed?

I am planning on using c++, opencl, avx-512

I would like to know what kind of Linux will be used so that I can sort any build/dynamic library issues ahead of time.

Daniel

**Comment by codepilot** [comment]
Is it possible to bump this?

**Comment by eddieh-xlnx** [comment, marked as answer]
Hi @codepilot. Sorry for not following up. I think it's safe to say that we're looking to target AMD 3rd Gen EPYC processors, which do not have AVX-512. We're also looking at AMD Instinct MI210 accelerator and will have support for OpenCL.

> I would like to know what kind of Linux will be used so that I can sort any build/dynamic library issues ahead of time.

We're still working on the details of this. The alpha submission deadline towards the end of December is also intended as an opportunity for contestants to test their setups to make sure that we can iron out all library issues.
